locale/English.json

```diff
@@ -4,7 +4,8 @@
       "Scan": "Scan",
       "Settings": "Settings",
       "Activity": "Activity",
-      "seqrByDhiway": "SEQR by DHIWAY"
+      "seqrByDhiway": "SEQR by DHIWAY",
+      "LanguagePage" : "Choose Language"
```

Comment:
couldn't we use `langChange` in Line 66?

Reply:
Can we use again langChange as a header??

Reply:
We can. Lets hear from others on what they think.